cmd/validators/main.go

```
@@ -120,12 +127,159 @@ type pubKeyInfo struct {
 	Value string `json:"value"`
 }
 
+func cmdMultisigNew(out io.Writer) func() error {
+	return func() error {
+		flag.Usage = func() {
+			fmt.Fprint(flag.CommandLine.Output(), `
+Create a new validator update request with multi signature authentication.
+Created request is binary serialized and written to standard output.
+
+Returned request must be signed by other parties before it can be submitted.
+
+`)
+			flag.PrintDefaults()
+		}
+		var (
+			pubKeyFl       = flag.String("pubkey", "", "Base64 encoded, ed25519 public key.")
+			multisigAddrFl = flag.String("multisig", "", "Address of the multisig contract that this request will authenticate with.")
+			powerFl        = flag.Int64("power", 10, "Validator node power. Set to 0 to delete a node.")
+		)
+		flag.Parse()
+
+		if *pubKeyFl == "" {
+			return errors.New("public key is required")
+		}
+		pubkey, err := base64.StdEncoding.DecodeString(*pubKeyFl)
+		if err != nil {
+			return fmt.Errorf("cannot base64 decode public key: %s", err)
+		}
+
+		if *multisigAddrFl == "" {
+			return errors.New("multisig address is required")
+		}
+
+		addValidatorTx := client.SetValidatorTx(
+			&validators.ValidatorUpdate{
+				Pubkey: validators.Pubkey{
+					Type: "ed25519",
+					Data: pubkey,
+				},
+				Power: *powerFl,
+			},
+		)
+
+		raw, err := addValidatorTx.Marshal()
+		if err != nil {
+			return fmt.Errorf("cannot serialize transaction: %s", err)
+		}
+		out.Write(raw)
```

pp: you could return the err instead of ignoring it.